### DISCUSSION TITLE
Public api request via postman: PUT  status 409

### DISCUSSION THREAD

**Discussion by Anthony69600** [Budibase/budibase#5705] [Help]
Hello everyone,

My app is in unpublished mode to make api calls
I'm trying to make public api calls on the budibase application locally. I manage very well to create a project and to recover this information.
On the other hand, I can't modify it (update), delete and retrieve it. I have a return {"message": "Document update conflict.", "status": 409 } I add screenshots to show my problem. 

I thank you in advance

![Capture d’écran de 2022-05-04 16-19-55](https://user-images.githubusercontent.com/76996157/166704093-95894f97-860a-4ebc-b33e-3331d110f8a8.png)

![Capture d’écran de 2022-05-04 16-19-55](https://user-images.githubusercontent.com/76996157/166703696-45c6187d-8e8b-490b-9a15-af0e8bea1347.png)

![Capture d’écran de 2022-05-04 16-20-04](https://user-images.githubusercontent.com/76996157/166703894-3806c902-497b-4daa-bbb4-e510b171c930.png)

![Capture d’écran de 2022-05-04 16-21-52](https://user-images.githubusercontent.com/76996157/166703980-d9b95d49-40a8-475a-8773-57987ffe2205.png)

**Comment by melohagan** [comment]
HI @Anthony69600 

I was able to reproduce this bug, and I've created an issue here: https://github.com/Budibase/budibase/issues/5709
